tests/__init__.py

```diff
@@ -0,0 +1,98 @@
+# Built-in Imports
+import os
+import sys
+
+# Cloudify Imports
+from ecosystem_tests import (
+    PasswordFilter,
+    EcosystemTestBase,
+    utils as eco_utils)
+
+
+class EnvironmentSetupTestBase(EcosystemTestBase):
+
+    def setUp(self):
+        if self.password not in self.sensitive_data:
+            self.sensitive_data.append(self.password)
+        sys.stdout = PasswordFilter(self.sensitive_data, sys.stdout)
+        sys.stderr = PasswordFilter(self.sensitive_data, sys.stderr)
+        self.cfy_local = self.setup_cfy_local()
+        if 'ECOSYSTEM_SESSION_MANAGER_IP' in os.environ:
+            self.manager_ip = \
+                os.environ['ECOSYSTEM_SESSION_MANAGER_IP']
+        else:
+            self.install_manager()
+            self.initialize_manager_profile()
```

Comment:
@EarthmanT Do we need to override this method ? cannot we depend on the base method ?

Reply:
I don't want to upload plugins for this test.

Reply:
make sense